### ISSUE TITLE
Get a real design, potentially use a CSS framework like SASS

### ISSUE THREAD

**Issue by marktheunissen** [vole/vole#36]
Ditch Twitter Bootstrap in favor of a custom design.

**Comment by rupl** [comment]
Do you have any preference for the Sass? I have a branch started and gave the indented syntax a go just to try something new. It basically just adds Sass but keeps the current design.

https://github.com/rupl/vole/compare/36-design

**Comment by marktheunissen** [comment]
Yeah I like SASS. What is the point of the syntax change? I think I prefer the css-like style.

**Comment by rupl** [comment]
I just wanted to try the other syntax because I never have. It's much more compact and it's whitespace-sensitive so I knew @pifantastic would like it :grin: 

Short example from my branch:

``` scss
// SCSS
.post .span6 img {
  @include transition(.3s all ease-in-out);
}

// Sass
.post .span6 img
  +transition(.3s all ease-in-out)
```

It's no trouble to switch. There's a conversion command provided with Sass. Shall I switch everything to SCSS?

**Comment by lasselicht** [comment]
I would be willing to make a redesign and maybe get some designers and programmers on board.

**Comment by rupl** [comment]
Have at it, my branch hasn't been updated since original convo.
